### PR TITLE
#51 #issue51 fixed issue 51 added empty sln when running the generato…

### PR DESCRIPTION
**Pull request by istern**
Solution file missing when running generator for empty helix solution

To create a valid starting point for a new developer creating a new helix solution, we need to create a empty visual solution file,

The pull request adds a new solution template to we need to maintain two sln template one for pentia tools and one for empty solutions

### CODE REVIEW

**Comment by Saturate** [comment]
If you wait until #52 has been merged, you can write tests for this, that ensures the content as well.

**Review by Hafeok — APPROVED** [review]
Looks good and nothing broken :) 👍

**Pull request merged**
